source/contact.erb

```diff
@@ -0,0 +1,104 @@
+---
+title: "get in touch..."
+subtitle: "to arrange a meeting or have a chat, just drop us a line"
+
+offices:
+
+  - name: "London"
+    address: |
+      17 Blossom Street
+      E1 6PL
+      London, UK
+    humanized_phone_number: "+44 20 7183 4250"
+    phone_number: "+442071834250"
+    email: "hello@unboxed.co"
+    map_link: "http://goo.gl/maps/AuKrs"
+    map_image: "https://a.tiles.mapbox.com/v4/web-unboxed.oebohej0/attribution,zoompan,zoomwheel,geocoder,share.html?access_token=pk.eyJ1Ijoid2ViLXVuYm94ZWQiLCJhIjoiY2lpN2hneWlvMDA4YnYzbHg2NHFqbW9yaSJ9.J1ksUtXP8Xsb2piQUcWJfA"
+
+    transport_directions: "nearest stations"
+    station_and_parking:
+      - name: "liverpool street"
+        link: "https://tfl.gov.uk/tube/stop/940GZZLULVT/liverpool-street-underground-station"
+        icon: "tfl-icon__underground.png"
+      - name: "shoreditch highstreet"
+        link: "https://tfl.gov.uk/overground/stop/910GSHRDHST/shoreditch-high-street-rail-station"
+        icon: "tfl-icon__overground.png"
+
+  - name: "Cape Town"
+    address: |
+      303 Buchanan Chambers
+      Cnr Pearce & Warwick Street
+      7735
+      Claremont, SA
+    humanized_phone_number: "+27 21 671 4503"
+    phone_number: "+27216714503"
+    email: "hello@unboxed.co"
+    map_link: "http://goo.gl/maps/JkNC9"
+    map_image: "https://a.tiles.mapbox.com/v4/web-unboxed.oec60hbb/attribution,zoompan,zoomwheel,geocoder,share.html?access_token=pk.eyJ1Ijoid2ViLXVuYm94ZWQiLCJhIjoiY2lpN2hneWlvMDA4YnYzbHg2NHFqbW9yaSJ9.J1ksUtXP8Xsb2piQUcWJfA"
+
+    transport_directions: "nearest parking and station"
+    station_and_parking:
+      - name: "grove avenue"
+        link: "https://www.parkme.com/map?lot=122336"
+        icon: "transport-icon__car.png"
+      - name: "claremont"
+        link: "http://www.capemetrorail.co.za/Route-Map.html"
+        icon: "transport-icon__train.png"
+---
+
+<header class="page-header">
+  <div class="page-header__container">
+    <h1 class="page-header__title">
+      <%= current_page.data.title %>
+    </h1>
+
+    <p class="page-header__subtitle">
+      <%= current_page.data.subtitle %>
+    </p>
+  </div>
+</header>
+
+<div class="container--contact-page">
```

Comment:
Another stylistic opinion but the elements below look very squashed up and more difficult to read than the above element - could we add some new lines between the different blocks?